src/core/core.js

```diff
@@ -154,7 +156,7 @@ export default class Core {
         queryTrigger: this.globalStorage.getState('queryTrigger'),
         sessionTrackingEnabled: this.globalStorage.getState(StorageKeys.SESSIONS_OPT_IN),
         sortBys: this.globalStorage.getState(StorageKeys.SORT_BYS),
-        locationRadius: this.filterRegistry.getLocationRadiusPayload()
+        locationRadius: locationRadiusFilterNode ? locationRadiusFilterNode.getFilter().value : null
```

Comment:
Just to confirm, is locationRadius: null the same thing as leaving out locationRadius on the back-end?

Reply:
yup, we have a data cleaning step that removes all null and undefined params